alerts/tahoma.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Tahoma integration incompatible with firmware 2020.3.4-12"
+created: 2020-07-03 12:00:00
+updated: 2020-07-03 12:00:00
+integrations:
+  - tahoma
+github_issue: https://github.com/home-assistant/core/issues/37291
+homeassistant: ">0.49"
+---
+
+Updating your Somfy TaHoma bridge 2020.3.4-12 will cause the Tahoma integration for Home Assistant to no longer work.
+
+## Workaround
+The Tahoma integration for Home Assistant is in the process of being completely rewritten and modernized. An initial beta version has been [released as a custom integration](https://github.com/imicknl/ha-tahoma) in order to gather feedback and bug reports. After the testing period concludes the changes will be integrated back into Home Assistant.
```

Comment:
Isn't the actual workaround to use the official Somfy integration?

Reply:
Unfortunately not, the official Somfy API only supports a few devices. However, would be good to mention that in this alert as well. The Somfy and new Somfy TaHoma integration could be used next to each other.

Reply:
@balloob updated the text to first steer people to the Somfy integration. Thanks!

Reply:
@tetienne @vlebourl any comments / additions?

Reply:
Can we not do a hotfix directly on the Tahoma component? I don't know if your fix was complex or not.